.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
```

Comment:
Does this really need a Dependabot config file?
Dependabot does create PRs for vulnerable dependencies right now without this config file, is there something extra we gain with it?

Reply:
Yes! It also creates pr for every new version of npm or github-actions dependencies